starfish/pipeline/features/spots/detector/gaussian.py

```diff
@@ -1,25 +1,29 @@
-from itertools import product
-from numbers import Number
-from typing import Tuple
+from typing import Union
 
 import numpy as np
 import pandas as pd
+import xarray as xr
 from skimage.feature import blob_log
 
-from starfish.constants import Indices, Features
+from starfish.constants import Features
 from starfish.image import ImageStack
-from starfish.munge import dataframe_to_multiindex
 from starfish.intensity_table import IntensityTable
+from starfish.pipeline.features.spot_attributes import SpotAttributes
+from starfish.pipeline.features.spots.detector.detect import (
+    measure_spot_intensity,
+    detect_spots,
+)
+from starfish.typing import Number
 from starfish.util.argparse import FsExistsType
 from ._base import SpotFinderAlgorithmBase
 
 
 class GaussianSpotDetector(SpotFinderAlgorithmBase):
 
     def __init__(
-            self, min_sigma, max_sigma, num_sigma, threshold,
-            blobs_stack, overlap=0.5, measurement_type='max', is_volume: bool=True, **kwargs
-    ) -> None:
+            self, min_sigma: Number, max_sigma: Number, num_sigma: int, threshold: Number,
+            overlap=0.5, measurement_type='max', is_volume: bool=True, **kwargs) \
+            -> None:
         """Multi-dimensional gaussian spot detector
```

Comment:
Add -- "... this is a wrapper for skimage.blahblahblah blah" or something like that, analagous to peak local max finder.